frontend/src/components/HomePage/Card.jsx

```diff
@@ -1,14 +1,42 @@
 import PropTypes from 'prop-types';
 
 import CategoryTag from '../Category/CategoryTag';
+import { Box } from '@mui/material';
+import { getSponsorMembers } from '../../utils';
+
+/** Bill meta info snippet */
+const SubInfo = ({ number, sponsors, status }) => (
+  // todo: color
+  <Box sx={{ color: '#6D6D6D' }}>
```

Comment:
is there a reason for using a mui Box and not just a div? (just curious)

Reply:
In this case, not necessary. I'm just used to using whatever framework is included.